tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -653,3 +656,134 @@ def test_ops_different_share_class(get_clients) -> None:
 def test_get_shape_none() -> None:
     with pytest.raises(ValueError):
         MPCTensor._get_shape("mul", None, None)
+
+
+@pytest.mark.parametrize("bit", [0, 1])
+def test_bin_public_xor(get_clients, bit) -> None:
+    clients = get_clients(3)
+    falcon = Protocol.registered_protocols["Falcon"]()
+    session = Session(parties=clients, protocol=falcon)
+    session.config = Config(encoder_base=1, encoder_precision=0)
+    SessionManager.setup_mpc(session)
+
+    x = torch.tensor([[1, 0], [0, 1]], dtype=torch.bool)
```

Comment:
MAYBE: We could remove torch.bool, as the test for public xor is done for session ring_size.